.jscpd.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ignore": ["dbt/models/**/*.sql"]
```

Comment:
These models necessarily involve duplication for now, so I'm ignoring them in the copy/paste linter.